feg/protos/s6a_proxy.proto

```diff
@@ -105,6 +105,8 @@ message AuthenticationInformationAnswer {
         bytes xres = 2;
         bytes autn = 3;
         bytes kasme = 4;
+        bytes ck = 5;
+        bytes ik = 6;
```

Comment:
According to https://www.etsi.org/deliver/etsi_ts/129200_129299/129272/15.10.00_60/ts_129272v151000p.pdf
CK & IK are not a part of E-UTRAN-Vector, they are in UTRAN-Vector (see below) & you can use already defined UTRANVector below.

Reply:
Actually, we need a bit more than reusing UTRANVector. I suggest adding an NGRANVector and return kausf (rather than kasme). This would require adding a new field `num_requested_ngran_vectors` to Authentication Information Request so that s6a proxy on subscriberdb knows that it needs to generate `kausf` and not `kasme` for the response. The logic to generate `kausf` needs to be added to subscriberdb. cc @emakeev @panyogesh

Reply:
what RFC/TS defines NGRANVector? S6a proxy converts gRPC fields to diameter AVPs and forwards requests to HSS. Just adding fields to protos won't be enough, we'd need to add the support for them in s6a_proxy & most likely - go-diameter. S6a proxy is dedicated to one specific protocol S6a (29.272), I checked up to v16.6 and I don't see  any mentioning NGRANVector there. s6a_proxy does not generate anything, it just passes through vectors returned by HSS & in general, doesn't depend or even know of subscriberdb existence.
It's possible that we may need a new service/RPC instead.

Reply:
@emakeev these are 5G extensions there is no HSS etc. Why should proto extensions break FEG?

Reply:
@ulaskozat, they shouldn't break FeG/s6a_proxy, we can add optional fields to AuthenticationInformationRequest/Answer they just won't be supported by FeG. It could be confusing to have 5G extensions defined in s6a_proxy.proto that's why I am suggesting to  consider different service or RPC

Reply:
@emakeev I understand the concern. I will ask the team to implement a 5G specific RPC call that can be extended independently.

Reply:
Not sure if you refer to this, but note we already have some 5g fields on s6a_proxy [here](http://github.com/magma/magma/blob/7441d3f1467d142e0f01f101c0e5c0a7a1228592/feg/protos/s6a_proxy.proto#L146-L146) and [here](http://github.com/magma/magma/blob/7441d3f1467d142e0f01f101c0e5c0a7a1228592/feg/protos/s6a_proxy.proto#L281-L281)


Those are on [3gpp 29.272](https://www.etsi.org/deliver/etsi_ts/129200_129299/129272/16.06.00_60/ts_129272v160600p.pdf)

Reply:
@uri200 this is a draft PR. Note that 5G fields you are referring to on s6a are for NSA that uses 4G core.